malebranche/client/logger/logger.py

```diff
@@ -0,0 +1,55 @@
+from __future__ import (
+    annotations,
+)
+
+import logging
+from logging import (
+    Handler,
+)
+
+from malebranche.client.context import (
+    ContextManager,
+)
+
+from .filter import (
+    MalebrancheLogFilter,
+)
+from .http_avro_handler import (
+    HttpAvroHandler,
+)
+
+
+class Logger:
+    def __init__(
+        self,
+        module: str,
+        context: ContextManager,
+        level=logging.DEBUG,
+        handler: Handler = HttpAvroHandler(host="localhost:5000", url="/logs", method="POST"),
+    ):
+        self.logger: logging.Logger = self._create_logger(module, context, level, handler)
+
+    def _create_logger(
```

Comment:
In order to avoid future mistakes, I recommend you to explicitly define this method as `classmethod` or`staticmethod`, as it is called during the initialization process and someone may be tempted to call another instance method before the instance is ready for that.